src/styles/card.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+  
+.card__price-value,
+.card__price-label,
+.card__title-label,
+.card_rate-label,
+.card__title {
+  margin: 0;
+  cursor: default;
+}
```

Comment:
why are you adding each class for this? if you want to reset styles, use tags like "div, p"

Reply:
i want to reset styles only for this classes, not for all div or p

Reply:
It is indeed better to do it for tags in smth like `reset.css`
What if you gonna have hundreds of classes for links in big project?